ampd/src/grpc/mod.rs

```diff
@@ -35,6 +36,8 @@ pub struct Config {
     pub port: u16,
     pub concurrency_limit: nonempty::Usize,
     pub concurrency_limit_per_connection: nonempty::Usize,
+    #[serde(with = "humantime_serde")]
+    pub reqs_timeout: Duration,
```

Comment:
```suggestion
    pub request_timeout: Duration,
```
as function parameter it's relatively easy to understand, here it should have a proper name because it's going to be converted into toml